code/game/objects/items/devices/geiger_counter.dm

```diff
@@ -111,7 +111,7 @@
 	loop.last_radiation = radiation_count
 	loop.start()
 
-/obj/item/geiger_counter/rad_act(amount)
+/obj/item/geiger_counter/rad_act(amount, collectable_radiation)
```

Comment:
Make geiger counters show the amount of collectable vs non-collectable radiation